### PR TITLE
No place left

### PR DESCRIPTION
**Pull request by m-heyda**

### CODE REVIEW

**Review by michalwarda — APPROVED** [review]
I like it. Good job!

File: src/components/Wrapper/style.scss [hunk 1]
```diff
@@ -1,26 +1,30 @@
 .wrapper {
   position: relative;
+  display: flex;
+  height: 100vh;
```
Comment (kingasieminiak) [inline src/components/Wrapper/style.scss:4]: 100vh and overflow hidden might caused some problems if there will be more text/ components in the view

File: src/routes/noplace/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  <div className={style.contentWrapper}>
+    <div className={style.splashScreen}>
+      <BackCar className={style.backCar} />
+      <GrayCar className={style.grayCar} />
+    </div>
+    <div>
+      <Heading color='Blue'>
+        Dziś nie ma wolnych miejsc
+      </Heading>
+      <p className={style.graySubheader}>
```
Comment (kingasieminiak) [inline src/routes/noplace/index.js:16]: You can use the `hint` component with a modifier.

Comment (m-heyda) [reply src/routes/noplace/index.js:16]: Yea, I will after merge with 'your-place-view' branch. Already refactored it so it's ready to use it here :)

File: src/components/Wrapper/style.scss (hunk 1 above)
Comment (m-heyda) [reply src/components/Wrapper/style.scss:4]: This is also fixed on another branch

**Pull request merged**